diff --git a/build.gradle b/build.gradle
index 4d446da..e6ce780 100644
--- a/build.gradle
+++ b/build.gradle
@@ -5,6 +5,7 @@ buildscript {
 plugins {
     id 'nebula.netflixoss' version '9.1.0'
     id 'nebula.dependency-lock' version '11.1.2'
+    id 'org.openrewrite.rewrite' version '5.40.0'
 }
 
 apply plugin: 'java'
@@ -18,6 +19,12 @@ mainClassName = 'com.netflix.search.query.QueryTests'
 repositories {
     mavenCentral()
 }
+
+rewrite {
+    activeRecipe(
+        'org.openrewrite.staticanalysis.CommonStaticAnalysis',
+    )
+}
   
 dependencies {
     compile 'com.google.apis:google-api-services-sheets:v4-rev531-1.23.0'
